src/llvm_backend.hpp

```diff
@@ -71,6 +66,7 @@ struct lbProcedure;
 struct lbValue {
 	LLVMValueRef value;
 	Type *type;
+	bool is_packed;
```

Comment:
This looks like a massive bodge. Can we not annotate it elsewhere?

Reply:
Removed in favor of your approach in https://github.com/odin-lang/Odin/commit/de5ce90fa70c8ae4b859b50704dd0ffacbeedb1e